Sources/BraintreeUIComponents/BTPaymentButtons.swift

```diff
@@ -0,0 +1,48 @@
+import SwiftUI
+
+struct PaymentButton: View {
```

Comment:
We should update this to `PayPalButton` and make the struct public and with docstrings. I think we are also missing an init with the color.